lectures/pandas.md

```diff
@@ -593,47 +590,39 @@ Note that pandas offers many other file type alternatives.
 
 Pandas has [a wide variety](https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html) of top-level methods that we can use to read, excel, json, parquet or plug straight into a database server.
 
-### Using {index}`pandas_datareader <single: pandas_datareader>` and {index}`yfinance <single: yfinance>` to Access Data
-
-```{index} single: Python; pandas-datareader
-```
+### Using {index}`wbgapi <single: wbgapi>` and {index}`yfinance <single: yfinance>` to Access Data
 
-The maker of pandas has also authored a library called
-[pandas_datareader](https://pandas-datareader.readthedocs.io/en/latest/) that
-gives programmatic access to many data sources straight from the Jupyter notebook.
+The [wbgapi](https://pypi.org/project/wbgapi/) python library can be used to fetch data from the many databases published by the World Bank.
 
-While some sources require an access key, many of the most important (e.g., FRED, [OECD](https://data.oecd.org/), [EUROSTAT](https://ec.europa.eu/eurostat/data/database) and the World Bank) are free to use.
+```{note}
+You can find some useful information about the [wbgapi](https://pypi.org/project/wbgapi/) package in this [world bank blog post](https://blogs.worldbank.org/en/opendata/introducing-wbgapi-new-python-package-accessing-world-bank-data), in addition to this [tutorial](https://github.com/tgherzog/wbgapi/blob/master/examples/wbgapi-quickstart.ipynb)
+```
 
 We will also use [yfinance](https://pypi.org/project/yfinance/) to fetch data from Yahoo finance
```

Comment:
@jstac sorry just merged this without a review. 

The only thing that is review worthy is the changes from `pandas_datareader` to `wbgapi`